release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3090,3 +3090,36 @@ link:https://access.redhat.com/solutions/6145452[{product-title} 4.6.36 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-6-38"]
+=== RHBA-2021:2641 - {product-title} 4.6.38 bug fix update
+
+Issued: 2021-07-14
+
+{product-title} release 4.6.38 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:2641[RHBA-2021:2641] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:2642[RHBA-2021:2642] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/2642[{product-title} 4.6.38 container image list]
+
+[id="ocp-4-6-38-bug-fixes"]
+==== Bug fixes
+
+* Previously, when starting a single-stack IPv6 cluster on nodes with IPv4 address, the kubelet might have used the IPv4 IP instead of the IPv6 IP for the node IP. Consequently, host network pods would have IPv4 IPs rather than IPv6 IPs, which made them unreachable from IPv6-only pods. This update fixes the node-IP-picking code, which results in the kubelet using the IPv6 IPs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942506[*BZ#1942506*])
```

Comment:
This isn't explicitly covered in the style guide, but the section about "Using addresses in examples" always pairs "address" with IPv4 and IPv6.

```suggestion
* Previously, when starting a single-stack IPv6 cluster on nodes with IPv4 address, the kubelet might have used the IPv4 address instead of the IPv6 address for the node IP address. Consequently, host network pods would have IPv4 addresses rather than IPv6 addresses, which made them unreachable from IPv6-only pods. This update fixes the node-IP-picking code, which results in the kubelet using the IPv6 addresses. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942506[*BZ#1942506*])
```